test/quantization/core/experimental/fx_graph_mode_apot.py

```diff
@@ -0,0 +1,272 @@
+calibrate(prepared_model1, data_loader_test)  # run calibration on sample data
+quantized_model1 = convert_fx(prepared_model1)  # convert the calibrated model to a quantized model
+
+top1, top5 = evaluate(quantized_model1, criterion, data_loader_test)
+print("Model #1 Evaluation accuracy on test dataset (b=4, k=2): %2.2f, %2.2f" % (top1.avg, top5.avg))
+
+"""
+Prepare model with uniform activation, APoT weight
+(b=8, k=2)
+"""
+
+qconfig2 = get_apot_weights_qconfig_8bit()
+qconfig_dict2 = {"object_type": [(torch.nn.Linear, qconfig2)]}
+prepared_model2 = prepare_fx(float_model, qconfig_dict2)  # fuse modules and insert observers
+calibrate(prepared_model2, data_loader_test)  # run calibration on sample data
+quantized_model2 = convert_fx(prepared_model2)  # convert the calibrated model to a quantized model
+
+top1, top5 = evaluate(quantized_model2, criterion, data_loader_test)
+print("Model #2 Evaluation accuracy on test dataset (b=8, k=2): %2.2f, %2.2f" % (top1.avg, top5.avg))
+
+"""
+Prepare model with uniform activation, APoT weight
+(b=4, k=2)
+"""
+
+qconfig2 = get_apot_weights_qconfig_4bit()
+qconfig_dict2 = {"object_type": [(torch.nn.Linear, qconfig2)]}
+prepared_model2 = prepare_fx(float_model, qconfig_dict2)  # fuse modules and insert observers
+calibrate(prepared_model2, data_loader_test)  # run calibration on sample data
+quantized_model2 = convert_fx(prepared_model2)  # convert the calibrated model to a quantized model
+
+top1, top5 = evaluate(quantized_model2, criterion, data_loader_test)
+print("Model #2 Evaluation accuracy on test dataset (b=4, k=2): %2.2f, %2.2f" % (top1.avg, top5.avg))
+
+
+"""
+Prepare model with APoT activation and weight
+(b=8, k=2)
+"""
+
+qconfig3 = get_apot_qconfig_8bit()
+qconfig_dict3 = {"object_type": [(torch.nn.Linear, qconfig3)]}
+prepared_model3 = prepare_fx(float_model, qconfig_dict3)  # fuse modules and insert observers
+calibrate(prepared_model3, data_loader_test)  # run calibration on sample data
+quantized_model3 = convert_fx(prepared_model3)  # convert the calibrated model to a quantized model
+
+top1, top5 = evaluate(quantized_model3, criterion, data_loader_test)
+print("Model #3 Evaluation accuracy on test dataset (b=8, k=2): %2.2f, %2.2f" % (top1.avg, top5.avg))
+
+"""
+Prepare model with APoT activation and weight
+(b=4, k=2)
+"""
+
+qconfig3 = get_apot_qconfig_4bit()
+qconfig_dict3 = {"object_type": [(torch.nn.Linear, qconfig3)]}
+prepared_model3 = prepare_fx(float_model, qconfig_dict3)  # fuse modules and insert observers
+calibrate(prepared_model3, data_loader_test)  # run calibration on sample data
+quantized_model3 = convert_fx(prepared_model3)  # convert the calibrated model to a quantized model
+
+top1, top5 = evaluate(quantized_model3, criterion, data_loader_test)
+print("Model #3 Evaluation accuracy on test dataset (b=4, k=2): %2.2f, %2.2f" % (top1.avg, top5.avg))
+
+"""
+Prepare eager mode quantized model
+"""
+
+from torchvision.models.quantization.resnet import resnet18
```

Comment:
would be good to put this to a separate function as well

Reply:
since this is the only instance of eager mode quantization maybe i will leave outside? not sure if a helper function would simplify much